its/src/org/sonarlint/eclipse/its/OpenInIdeTest.java

```diff
@@ -196,25 +261,27 @@ public void test_open_in_ide_when_project_already_bound() throws IOException, In
     } catch (Exception ignored) {
     }
 
-    // 3) delete file
+    // 2) delete file
     rootProject.getResource("src/main/java", "taint", "Variables.java").delete();
 
-    // 4) get S1481 issue key / branch name from SonarQube
+    // 3) get S1481 issue key / branch name from SonarQube
     var s1481 = getFirstIssue(S1481);
     assertThat(s1481).isNotNull();
 
     var branch = getFirstBranch();
     assertThat(branch).isNotNull();
 
-    // 5) trigger "Open in IDE" feature: issue not found because file not found
+    // 4) trigger "Open in IDE" feature: issue not found because file not found
     // -> because the API is quite slow we have to await the opening of the error dialog
     triggerOpenInIDE(orchestrator.getServer().getUrl(), branch.getName(), s1481.getKey());
     new WaitUntil(new OpenInIdeDialogOpened(), TimePeriod.DEFAULT);
     new OpenInIdeDialog().ok();
 
-    // 6) change preferences regarding new code / issue filter + change file content
+    // 5) change preferences regarding new code / issue filter + change file content
     setNewCodePreference(IssuePeriod.NEW_CODE);
-    openFileAndWaitForAnalysisCompletion(rootProject.getResource("src/main/java", "taint", "SysOut.java"));
+    
+    rootProject.getResource("src/main/java", "taint", "SysOut.java").delete();
+    createClass("taint", "SysOut.java");
```

Comment:
FYI: Doing it this way as the other option was too flaky and broke easily.

Reply:
Maybe add a comment then

Reply:
Done